projects/plugins/boost/app/features/guide/src/ImageObserver.ts

```diff
@@ -0,0 +1,86 @@
+type ImageCallback = ( images: HTMLElement[] ) => void;
+
+const onChange = ( callback: ImageCallback ): MutationCallback => mutationList => {
+	const mutations = mutationList
+		.filter( mutation => {
+			// Ignore mutations that added `jetpack-boost-guide-image` class
+			if ( mutation.type === 'attributes' && mutation.attributeName === 'class' ) {
+				const target = mutation.target as HTMLImageElement;
+				if ( target.classList.contains( 'jetpack-boost-guide-image' ) ) {
```

Comment:
Yeah, the mutation observer is a quick hack that I threw together, I actually think it may not need the observer at the moment, so for MVP I might just disable this entirely until it's cleaned up significantly.

Reply:
Disabled mutationObserver for now